lib/web/ui/integration.go

```diff
@@ -224,3 +224,20 @@ type AWSOIDCListEC2ICEResponse struct {
 	// If non-empty, it can be used to request the next page.
 	NextToken string `json:"nextToken,omitempty"`
 }
+
+// AWSOIDCDeployEC2ICERequest is a request to create an AWS EC2 Instance Connect Endpoint.
+type AWSOIDCDeployEC2ICERequest struct {
```

Comment:
Do these types need to be in the `ui` package? they're only ever used in the `web` package.

Reply:
I'm following the existing pattern, seems to be a convention

I think every type in `lib/web/ui` is only being used in `lib/web` package.